test/crypto/sign.js

```diff
@@ -42,6 +44,7 @@ describe('sign', () => {
 	const defaultSecondSecret = 'second secret';
 	const defaultPublicKey = '5d036a858ce89f844491762eb89e2bfbd50a4a0a0da658e4b2628b25b117ae09';
 	const defaultSecondPublicKey = '0401c8ac9f29ded9e1e4d5b6b43051cb25b22f27c7b7b35092161e851946f82f';
+	const firstSecret = '123';
```

Comment:
Why do we need this as well as `defaultSecret`?

Reply:
This is used in signTransaction and verify . Moved to the top no relevance for this PR

Reply:
But why not use `defaultSecret` for all of these instead since we're refactoring anyway?